<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionSetupDialog</class>
 <widget class="QDialog" name="PositionSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>840</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Position setup</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topLayout">
     <item>
      <layout class="QVBoxLayout" name="boardColumn">
       <item>
        <widget class="QGroupBox" name="m_boardGroup">
         <property name="title">
          <string>Board</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableWidget" name="m_boardTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>320</width>
              <height>320</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::NoSelection</enum>
            </property>
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="rowCount">
             <number>8</number>
            </property>
            <property name="columnCount">
             <number>8</number>
            </property>
            <attribute name="horizontalHeaderVisible">
             <bool>false</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="pieceLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Selected piece:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="m_pieceSelector"/>
         </item>
         <item>
          <widget class="QPushButton" name="m_clearSquareBtn">
           <property name="text">
            <string>Clear square</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="boardToolsLayout">
         <item>
          <widget class="QPushButton" name="m_resetBoardBtn">
           <property name="text">
            <string>Reset to start</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="m_clearBoardBtn">
           <property name="text">
            <string>Clear board</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="m_gameInfoGroup">
         <property name="title">
          <string>Game information</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="labelAlignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Side to move:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="sideLayout">
            <item>
             <widget class="QRadioButton" name="m_whiteToMoveRadio">
              <property name="text">
               <string>White</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="m_blackToMoveRadio">
              <property name="text">
               <string>Black</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Castling:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="castlingLayout">
            <item>
             <widget class="QCheckBox" name="m_castleK">
              <property name="text">
               <string>White O-O</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_castleQ">
              <property name="text">
               <string>White O-O-O</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_castlek">
              <property name="text">
               <string>Black O-O</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="m_castleq">
              <property name="text">
               <string>Black O-O-O</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>En passant:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="m_enPassantEdit">
            <property name="placeholderText">
             <string>- or square (e.g. e3)</string>
            </property>
            <property name="maxLength">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Halfmove clock:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="m_halfMoveSpin">
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Fullmove number:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="m_fullMoveSpin">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="m_referenceGroup">
       <property name="title">
        <string>Reference image</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
          <widget class="QLabel" name="m_referenceImageLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="text">
            <string notr="true">No image loaded</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="imageButtonsLayout">
          <item>
           <widget class="QPushButton" name="m_loadImageBtn">
            <property name="text">
             <string>Load imageâ€¦</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_clearImageBtn">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="m_fenGroup">
     <property name="title">
      <string>FEN</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLineEdit" name="m_fenLineEdit"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="fenButtonsLayout">
        <item>
         <widget class="QPushButton" name="m_copyFenBtn">
          <property name="text">
           <string>Copy</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_pasteFenBtn">
          <property name="text">
           <string>Paste</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="m_applyFenBtn">
          <property name="text">
           <string>Apply from text</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="fenButtonsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PositionSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>597</y>
    </hint>
    <hint type="destinationlabel">
     <x>373</x>
     <y>628</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PositionSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>682</x>
     <y>603</y>
    </hint>
    <hint type="destinationlabel">
     <x>750</x>
     <y>630</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>



